src/rateEngine/RateCalculator.ts

```diff
@@ -1,17 +1,16 @@
-import RateElement from './RateElement';
-import LoadProfile from './LoadProfile';
 import sum from 'lodash/sum';
-import type { RateCalculatorInterface, RateElementInterface, RateElementFilterArgs } from './types';
+import RateElement from './RateElement';
+import type { RateCalculatorInterface, RateElementFilterArgs } from './types';
 
 class RateCalculator {
   private _rateElements: Array<RateElement>;
   utilityName: string | undefined;
   applicability: string | undefined;
   minimumBillAmount: number | undefined;
-  static shouldValidate: Boolean = true;
-  static shouldLogValidationErrors: Boolean = true;
+  static shouldValidate = true;
+  static shouldLogValidationErrors = true;
 
-  constructor({ name, utilityName, applicability, minimumBillAmount, rateElements, loadProfile }: RateCalculatorInterface) {
+  constructor({ name: _name, utilityName, applicability, minimumBillAmount, rateElements, loadProfile }: RateCalculatorInterface) {
```

Comment:
it's probably just an oversight that `name` isn't exposed publicly... can you just treat it like `utilityName`?

Reply:
Actually, it looks like we spread the rate's name onto the calculator in practice:

https://github.com/bellawatt/electric-rate-engine?tab=readme-ov-file#ratecalculator

(it's also possible that the README example doesn't adhere to the newly-exported RateCalculatorInterface)